x-pack/plugins/ml/public/data_frame/components/job_create/job_create_form.tsx

```diff
@@ -162,52 +180,203 @@ export const JobCreateForm: SFC<Props> = React.memo(
       }
     };
 
+    if (started === true && progressPercentComplete === undefined) {
+      // wrapping in function so we can keep the interval id in local scope
+      function startProgressBar() {
+        const interval = setInterval(async () => {
+          try {
+            const stats = await ml.dataFrame.getDataFrameTransformsStats(jobId);
+            const percent = Math.round(stats.transforms[0].state.progress.percent_complete);
+            setProgressPercentComplete(percent);
+            if (percent >= 100) {
```

Comment:
is it possible that a different user could stop the job before reaching 100% and so this loop would continue running?
Could this check also look for the state to see if the job is "running" (or whatever the state is)

Reply:
Good point! Thinking about this further, another user could as well start the job again, but then we'd have no way of knowing that and whether to start again. So I'd like to keep it as is, ok with you?

Reply:
I think ideally, when checking for the progress, if the job becomes "stopped" before reaching 100%, the UI should notify the user and stop polling the server.
But i don't think the current behaviour is particularly bad, and that scenario is unlikely.
I'm fine leaving it as is.